common/class.tx_dlf_document.php

```diff
@@ -119,6 +119,14 @@ final class tx_dlf_document {
      */
     protected $hasFulltext = FALSE;
 
+    /**
+     * Mimetype of fulltext application/tei+xml or text/xml?
+     *
+     * @var string
+     * @access protected
+     */
+    protected $mimetypeFulltext = '';
```

Comment:
This needs a getter method `_getMimetypeFulltext()` as well in order for plugins to get the fulltext's MIMETYPE.